backends/platform/ios7/ios7_osys_main.cpp

```diff
@@ -280,49 +282,64 @@ void OSystem_iOS7::saveState() {
 	if (g_engine && g_engine->hasFeature(Engine::kSupportsSavingDuringRuntime) && g_engine->canSaveAutosaveCurrently()) {
 		Common::String targetName(ConfMan.getActiveDomainName());
 		int saveSlot = g_engine->getAutosaveSlot();
+		if (saveSlot == -1)
+			return;
 		// Make sure we do not overwrite a user save
 		SaveStateDescriptor desc = g_engine->getMetaEngine()->querySaveMetaInfos(targetName.c_str(), saveSlot);
 		if (desc.getSaveSlot() != -1 && !desc.isAutosave())
 			return;
 
-		// Do the auto-save, and if successful store this it in the config
+		// Do the auto-save, and if successful store this it in the uset settings
```

Comment:
"uset" should be user I guess

Reply:
Right. But that edit was actually done in a first pass in which I considered using `NSUserDefaults`, and I ended up not doing that. So `user settings` isn't accurate either. I will change that.